os.ts

```diff
@@ -63,3 +75,64 @@ export function writeFileSync(
     writeFileSyncPerm: perm
   });
 }
+
+const readFileRequests = new Map<number, ReadFileRequest>();
+
+// TODO replace with bigint
+let nextReadFileRequestId = 0;
```

Comment:
or use `UUID` of some sort?

Reply:
Still have possibility of collision, though seems better for reusing if old one is released. Will see if I could introduce a UUID implementation.

Reply:
Actually, I could just probably simple generate a random number below certain size (so I could easily pass to Go) and simply check if the id is already occupied and regenerate